File: PaymentSDK/UIHostObjCAppUITests/TestHelper.m
//
//  TestHelper.m
//  UIHostObjCAppUITests
//
//  Created by Rahul Dhuri on 11/06/19.
//  Copyright © 2019 Network International. All rights reserved.
//

#import "TestHelper.h"

@implementation TestHelper

+(void)type:(NSString *)string in:(XCUIApplication *)app {
    for (int i = 0; i < string.length; i++) {
        unichar c = [string characterAtIndex: i];
        NSString *str = [NSString stringWithFormat: @"%c", c];
        [[app.keys objectForKeyedSubscript: str] tap];
    }
}

@end

File: PaymentSDK/UIHostAppUITests/TestHelper.swift
//
//  TestHelper.swift
//  UIHostAppUITests
//
//  Created by Niraj Chauhan on 5/5/19.
//  Copyright © 2019 Network International. All rights reserved.
//

import XCTest


class TestHelper {
    
    static func type(_ string: String, in app: XCUIApplication){
        for char in string {
            app.keys[String(char)].tap()
            
        }
    }
    
    static func delete(in app: XCUIApplication){
        app.keys["Delete"].tap()
    }
    
}

